### ISSUE TITLE
Json marshal Custom field of struct BlockHeader doesn't comply to fullnode

### ISSUE THREAD

**Issue by wanliqun** [Conflux-Chain/go-conflux-sdk#113]
```
testJson1 := `{"hash":"0xa6528367a9287ed3a66fc64457db15e2aaa93104a3fd06d4f0a2beb6cc1f26c8","parentHash":"0x5aef321e4e49f430ad6322af8a0133eae83e635f7893c996eb127dcf24a00b14","height":"0x792776","miner":"cfx:aatxetsp0kdarpdb5stdyex11dr3x6sb0jw2gykec0","deferredStateRoot":"0xa979a8c492c44a512aa9529911a7862e1b61ce2aa441645e865def9219d2c68b","deferredReceiptsRoot":"0xd5f7e7960e9b56753868260c280746c01353dcd1b91a20cee2c919d0dc7bf78b","deferredLogsBloomHash":"0xd397b3b043d87fcd6fad1291ff0bfd16401c274896d8c63a923727f077b8e0b5","blame":"0x0","transactionsRoot":"0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470","epochNumber":"0x792778","blockNumber":"0xf7cf1c","gasLimit":"0x1c9c380","gasUsed":"0x0","timestamp":"0x6026478e","difficulty":"0xa8b175a4dc","powQuality":"0x2223e36adc5","refereeHashes":["0x4e4fca2593068b1dc83ecae3c1eaf0e4d41623985fd03d7f15fc1d63f653e7d2"],"adaptive":false,"nonce":"0x209fc5fbe719dace","size":"0x0","custom":[[1]],"transactions":[]}`

	for _, bsJson := range []string{testJson1} {
		var bs BlockSummary
		err := json.Unmarshal([]byte(bsJson), &bs)
		fatalIfErr(t, err)

		jbytes, err := json.Marshal(bs)
		fatalIfErr(t, err)
		t.Log(string(jbytes)) // outputted below

		var bs2 BlockSummary
		err = json.Unmarshal(jbytes, &bs2)
		fatalIfErr(t, err)
	}
}

```

The custom field is json marshalled into ["AQ=="] rather than [[1]]:

{"hash":"0xa6528367a9287ed3a66fc64457db15e2aaa93104a3fd06d4f0a2beb6cc1f26c8","parentHash":"0x5aef321e4e49f430ad6322af8a0133eae83e635f7893c996eb127dcf24a00b14","height":"0x792776","miner":"cfx:aatxetsp0kdarpdb5stdyex11dr3x6sb0jw2gykec0","deferredStateRoot":"0xa979a8c492c44a512aa9529911a7862e1b61ce2aa441645e865def9219d2c68b","deferredReceiptsRoot":"0xd5f7e7960e9b56753868260c280746c01353dcd1b91a20cee2c919d0dc7bf78b","deferredLogsBloomHash":"0xd397b3b043d87fcd6fad1291ff0bfd16401c274896d8c63a923727f077b8e0b5","blame":"0x0","transactionsRoot":"0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470","epochNumber":"0x792778","blockNumber":"0xf7cf1c","gasLimit":"0x1c9c380","gasUsed":"0x0","timestamp":"0x6026478e","difficulty":"0xa8b175a4dc","powQuality":"0x2223e36adc5","refereeHashes":["0x4e4fca2593068b1dc83ecae3c1eaf0e4d41623985fd03d7f15fc1d63f653e7d2"],"adaptive":false,"nonce":"0x209fc5fbe719dace","size":"0x0","custom":["AQ=="],"transactions":[]}

**Comment by wangdayong228** [comment]
fixed in #144

**Comment by wangdayong228** [comment]
update in #148
